theme/less/components/read-more.less

```diff
@@ -0,0 +1,34 @@
+.read-more {
+  position: relative;
+  max-height: 284px;
+  overflow: hidden;
+  transition:max-height 0.15s ease-out;
+}
+
+.read-more.expand {
+  max-height: 5000px !important;
+  transition:max-height 0.35s ease-in-out;
+}
+
+.read-more-link {
+  --gradient-color: var(--w);
+  position: absolute;
+  bottom: 0;
+  width:100%;
+  height: 60px;
+  padding-top:40px;
```

Comment:
Could we have a bit more spacing between the fading description and the read more link?
See:
![image](https://user-images.githubusercontent.com/28541902/136769995-f880408c-02bb-45fe-90ab-5d2cd1a6a796.png)